app/src/main/java/com/dashxdemo/app/feature/login/LoginActivity.kt

```diff
@@ -14,6 +15,9 @@ class LoginActivity : AppCompatActivity() {
         binding = ActivityLoginBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
+        val window = this.window
+        window.statusBarColor = this.resources.getColor(R.color.black)
```

Comment:
This should be done in the theme XML rather than having programmatically, right? @puliajay43 @kiranattilicho